projects/crypto-currency/iluminados/app/src/components/Coinoftheday/index.jsx

```diff
@@ -19,7 +19,7 @@ function CoinOfTheDay() {
   },
   )
 
-  const day = () => {
+  const rendercoin = () => {
```

Comment:
Segunda palavra deve começar em maiusculo, `rendercoin` -> `renderCoin`
```suggestion
  const renderCoin = () => {
```